cypress/integration/createNewUser.js

```diff
@@ -0,0 +1,47 @@
+/// <reference types="cypress" />
+import { faker } from "@faker-js/faker";
+
+describe("Testing for add new customer ", () => {
+  before(function () {
+    cy.visitLoginPage();
+    cy.get("a.nav-link").last().click();
+    cy.login();
+  });
+
+  beforeEach(function () {
+    cy.preserveCookies();
+  });
+  const name = faker.name.fullName();
+  const company = faker.company.bs();
+  const address = faker.address.streetAddress(true);
+  const phone = faker.phone.number("984######");
+  const email = faker.internet.email();
+  const city = faker.address.city();
+  const data = [name, company, address, city, phone, email];
+
+  context("form fill for create new user of ITERA", () => {
+    beforeEach(() => {
+      cy.get(".btn-primary").click();
+      cy.get("#Name").type(name);
+      cy.get("#Company").type(company);
+      cy.get("#Address").type(address);
+      cy.get("#City").type(city);
+      cy.get("#Phone").type(phone);
+      cy.get("#Email").type(email);
+    });
+
+    it("should validate 'BACK TO LIST' button is working", () => {
```

Comment:
Please use the test description according to the test events inside the test block